process_jartic_archives.py

```
@@ -0,0 +1,280 @@
+                            'parameter': sensor.parameter.value,
+                            'value': str(measurement.value),
+                            'unit': sensor.unit.value
+                        })
+                        
+                        # Yield batch every 1000 measurements
+                        if len(measurements) >= 1000:
+                            yield archive_index, month_str, measurements, location_count, len(locations)
+                            measurements = []
+                        
+                except Exception as e:
+                    pass  # Skip sensor errors
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)